exercises/rotational-cipher/example.rs

```diff
@@ -0,0 +1,16 @@
+pub fn rotate(text: str, shift_key: u8) -> String {
```

Comment:
with `str`, it wouldn't compile: https://play.rust-lang.org/?gist=5ca113a1aed9629f2fc22114f3b55015&version=stable&backtrace=0

with `&str`, it would: https://play.rust-lang.org/?gist=3ac4315e213c2b756bdfba34ebea828c&version=stable&backtrace=0